packages/mask/src/plugins/Trader/SNSAdaptor/trending/TrendingView.tsx

```diff
@@ -336,13 +338,13 @@ export function TrendingView(props: TrendingViewProps) {
                                     : undefined,
                                 defaultOutputCoin: trending.coin
                                     ? createFungibleToken(
-                                          trending.coin.chainId ?? chainId,
-                                          isNativeTokenAddress(trending.coin.address)
+                                          swapExpectedContract.chainId ?? chainId,
+                                          isNativeTokenAddress(swapExpectedContract.address)
                                               ? SchemaType.Native
                                               : SchemaType.ERC20,
-                                          trending.coin.address ?? trending.coin.contract_address ?? '',
-                                          trending.coin.name,
-                                          trending.coin.symbol,
+                                          swapExpectedContract.address,
+                                          '',
```

Comment:
Better use the real coin name and symbol.

Reply:
The contract in the coin is not current chain

Reply:
Does this meet expectations?

Reply:
yes